aana/processors/speaker.py

```diff
@@ -0,0 +1,621 @@
+from collections import defaultdict
+
+from aana.core.models.asr import (
+    AsrSegment,
+    AsrWord,
+)
+from aana.core.models.speaker import SpeakerDiarizationSegment
+from aana.core.models.time import TimeInterval
+
+# Define sentence ending punctuations to split segments at sentence endings:
+sentence_ending_punctuations = ".?!"
+
+
+class ASRPostProcessingForDiarization:
```

Comment:
Just a small stylistic nitpick but we usually capitalise only first letter like `AsrPostProcessingForDiarization` instead of `ASRPostProcessingForDiarization`.

Reply:
renamed to `PostProcessingForDiarizedAsr` as it is more meaningful.